src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -289,19 +291,19 @@ const ActionSidebarContent: FC<ActionSidebarContentProps> = ({
               (error.message as unknown) === 'Auth failed' ||
               (error.code as unknown) === 4001
             ) {
-              setShowApolloNetworkError(true);
-            }
-
-            // Mutation failed in saga
-            if (error instanceof ApolloError) {
+              setCustomError({ message: formatText(MSG.apolloNetworkError) });
+            } else if (error instanceof ApolloError) {
+              // Mutation failed in saga
               const { networkError } = error;
 
               const statusCode = (networkError as { statusCode?: number })
                 ?.statusCode;
 
               if (statusCode === 403) {
-                setShowApolloNetworkError(true);
+                setCustomError({ message: formatText(MSG.apolloNetworkError) });
               }
+            } else if (error.arbitraryTxActionFailed) {
+              setCustomError({ message: formatText(MSG.arbitraryTxError) });
```

Comment:
Thanks for your refactoring @Nortsova 💯 

Now we no longer override the error messages 🙌 

I would propose to use `return` statements if a condition is met instead of `if-else` statements as it improves readability, but this is not a show-stopper for merging this PR ☀️